docs/api/helpers.md

```diff
@@ -0,0 +1,5 @@
+::: kloppy.domain.services.state_builder.add_state
+
+::: kloppy.helpers.to_pandas
+
+::: kloppy.helpers.transform
```

Comment:
@koenvo it seems like helpers.md is not included in _mkdocs.yml_, is that intentional?

Reply:
Was intentional but confusing :-)

Changed it a bit